the_debuginator.h

```diff
@@ -1304,11 +1317,38 @@ bool debuginator__distance_to_hot_item(DebuginatorItem* item, DebuginatorItem* h
 
 			child = debuginator__next_visible_sibling(child);
 		}
+	} else if (item->leaf.is_expanded && !item->is_filtered) {
+		*distance += item_height * item->leaf.num_values;
+		*distance += item->leaf.description_line_count * item_height;
 	}
-
 	return false;
 }
 
+int debuginator_total_height(TheDebuginator* debuginator) {
+	int height = 0;
+	DebuginatorItem *last_item = debuginator__first_visible_child(debuginator->root);
+	while (last_item && (last_item->next_sibling || last_item->is_folder)) {
+		if (last_item->next_sibling) {
+			last_item = debuginator__next_visible_sibling(last_item);
+		} else {
```

Comment:
Minor nitpicking :) For the debuginator I put the else { on a new line (so not } else { ).